specification/serialconsole/resource-manager/Microsoft.SerialConsole/stable/2018-05-01/examples/DisableConsoleExamples.json

```diff
@@ -7,7 +7,9 @@
   "responses": {
     "200": {
       "body": {
-        "disabled": true
+        "properties": {
```

Comment:
Please check model validation errors, these are coming due to adding this "properties" here.

Reply:
The 2018 rest api specs that were published were incorrect. The PRs align the current response with the documented one. The changes made in this PR to the 2018 spec intentionally lead to model validation errors